bindings/kepler.gl-jupyter/keplergl/keplergl.py

```diff
@@ -142,6 +143,24 @@ def add_data(self, data, name="unnamed"):
 
         self.data = copy
 
+    def show(self, data=None, config=None, read_only=False, center_map=False):
```

Comment:
Can you add this function to the docs?

Reply:
https://github.com/keplergl/kepler.gl/blob/master/docs/keplergl-jupyter/README.md

Reply:
Thanks, @heshan0131 Just updated the doc. Let me know any other issues.